### ISSUE TITLE
[Join Our Newsletter] Add Floating Message once clicked to allow people to enter their name

### ISSUE THREAD

**Issue by pelsasser** [MARKETProtocol/website#24]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
We should capture user's first and last names that want to join our newsletter

### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:It will help us direct marketing and email to them to avoid spam filters, etc

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: feature

### Current Behavior
[comment]: # (Describe what actually happened.)
Clicking the Arrow button on "Join our newsletter" subscribes people automatically

### Desired Behavior
[comment]: # (Describe what you expected to happen.)
A second screen / pop up that records their first and last name and then submits to GetResponse

## Before you `start work`
If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

**Comment by vikaskyadav** [comment]
Can't we integrate this newsletter subscription logic with services like MailChimp?

**Comment by pelsasser** [comment]
Mail chimp is closing down accounts of crypto projects, currently people have recommended GetResponse, although at the moment I am not thrilled with them.

**Comment by vikaskyadav** [comment]
oh yeah, I heard of this a few weeks back. Let us see what we can use for long term hasslefree.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.05 ETH (34.04 USD @ $680.72/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/MARKETProtocol/website/24/337).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $8,085.62 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.

 1. [@perfectmak](https://gitcoin.co/profile/perfectmak) 

 has committed to working on this project to be completed 6 days, 23 hours from now.

**Comment by dreday966** [comment]
hi, I have completed the work, but I need some ether to submit my work.

**Comment by eswarasai** [comment]
@dreday966 -- You can use [Gitcoin Faucet](https://gitcoin.co/faucet) to get some ETH. Moreover, you can raise a PR even without clicking the `Submit Work` button on Gitcoin, so that we can review it in the meantime while you get some ETH to update the status of the bounty on Gitcoin.

FYI, ideally this bounty belongs to @perfectmak since he called dibs on it. So, before starting work, please give a heads up on the bounty, so that everyone is under the same page.

@perfectmak -- I hope you haven't started work on this yet and cool with @dreday966 claiming the bounty since he had already worked on the task. Thanks.

**Comment by perfectmak** [comment]
It is cool. I haven't done tangible work on it. So he is free to claim it.

**Comment by dreday966** [comment]
@perfectmak thx, I just try to taste using gitcoin.

**Comment by gitcoinbot** [comment]
@perfectmak Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] auto removal (6 days)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@perfectmak has been removed for inactivity and [the issue](https://gitcoin.co/issue/MARKETProtocol/website/24/337) has been returned to an ‘Open’ Status. Let us know if you believe this has been done in error!

* [x] warning (3 days)
* [x] auto removal (6 days)

**Comment by pelsasser** [comment]
@dreday966 - please `submit work` on gitcoin so I can pay out the bounty

**Comment by dreday966** [comment]
@pelsasser My FAUCET request haven't be approved. Could you please just send ether to my wallet, I will close the work, thanks.

**Comment by eswarasai** [comment]
@dreday966 -- Can you please DM your ETH wallet address to @pelsasser on [MARKET Protocol Discord](https://marketprotocol.io/discord)? Thanks!

**Comment by dreday966** [comment]
@eswarasai ok, I have sent the address to @pelsasser.

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 0.05 ETH (38.16 USD @ $763.18/ETH) attached to this issue has been **cancelled** by the bounty submitter__

 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $8,052.18 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by pelsasser** [comment]
Sent directly. 
https://etherscan.io/tx/0xd05c2e6595f8caf1005cd216c06e6568c4cf79f38beb7a7b67885608966ad1a6

**Comment by weberV** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__This issue now has a funding of 0.05 ETH (34.61 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/MARKETProtocol/website/24/337).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $9,677.96 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
